docs/core/diagnostics/debug-highcpu.md

```diff
@@ -83,10 +83,10 @@ With the web app running, immediately after startup, the CPU isn't being consume
 
 `https://localhost:5001/api/diagscenario/highcpu/60000`
 
-Now, rerun the [dotnet-counters](dotnet-counters.md) command. To monitor just the `cpu-usage`, specify `System.Runtime[cpu-usage]` as part of the command.
+Now, rerun the [dotnet-counters](dotnet-counters.md) command. To monitor just the `cpu-usage`, add --counters `System.Runtime[cpu-usage]` as part of the command.
 
 ```dotnetcli
-dotnet-counters monitor --counters System.Runtime[cpu-usage] -p 22884 --refresh-interval 1
+dotnet-counters monitor -p 22884 --refresh-interval 1
```

Comment:
```suggestion
dotnet-counters monitor -p 22884 --refresh-interval 1 --counters System.Runtime[cpu-usage]
```
Otherwise it's the same command as before. And as the text states "add" put it at the end -- not in the middle as the previous code was (left side of diff).